src/client/opamClient.ml

```diff
@@ -996,6 +996,7 @@ let windows_checks ?cygwin_setup ?git_location config =
              let cygcheck =
                match setup with
                | `internal -> install_cygwin_tools ()
+               | `internal_with_pkgs _ -> install_cygwin_tools ()
```

Comment:
I believe the arguments should be actually used here, otherwise they are never used anywhere and the option doesn't do anything besides enabling internal cygwin

Reply:
Of course it is here.